CONTRIBUTING.md

```diff
@@ -89,6 +90,13 @@ When writing a new test for ES6, it is only necessary to include this tag when t
 
 This tag identifies the portion of the ECMAScript 6 standard that is tested by this test.
```

Comment:
The new `id` is an anchor in the spec whereas the old ones are section numbers. I think we should update the old ones' descriptions to call out the difference.